Northstar.CustomServers/mod/scripts/vscripts/mp/_gamestate_mp.nut

```diff
@@ -184,11 +184,17 @@ void function GameStateEnter_PickLoadout_Threaded()
 	// titan selection menu can change minPickLoadOutTime so we need to wait manually until we hit the time
 	while ( Time() < GetServerVar( "minPickLoadOutTime" ) )
 		WaitFrame()
-	
+
+	//added by Khalmee for the sake of fixing LTS spectator mode
+	foreach(player in GetPlayerArray()){
+		if(IsPrivateMatchSpectator(player))
+		InitialisePrivateMatchSpectatorPlayer( player )
+	}
```

Comment:
Formatting
```suggestion
	foreach ( player in GetPlayerArray() ){
		if ( IsPrivateMatchSpectator( player ) )
		InitialisePrivateMatchSpectatorPlayer( player )
	}
```

Reply:
Wait, why can I not commit suggestion myself? :c

Reply:
just invited you to the fork repo, should be able to commit suggestions now